mantidproject/mantid
# Replace celebration splash screen

# DO NOT MERGE THIS UNTIL r3.12 (4.0?)

Replaced the celebration splash screen with a normal one.

closes #19897

**To test:**


1. Startup Mantidplot, look at the splash screen
1. See a lack of Balloons, feel sad
1. Check that the version label at the bottom of the splash screen does not run over the image.


**Release Notes** 

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
